### PR TITLE
1.15 update (required changes)

### PR DESCRIPTION
**Pull request by Linguardium**
Loom updated to 2.7
updated Yarn+MC version
updated trinkets
fixed 2 lines of code

probably could update loader_version
would need a version bump

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,8 +3,8 @@ package_group=info.tehnut.armordisguise
 mod_version=1.0.1
 curse_id=347849
 
-mc_version=1.14.4
-mappings_version=14
+mc_version=1.15.2
+mappings_version=15
 loader_version=0.6.3+build.167
```
Comment (Linguardium) [inline gradle.properties]: loader could be updated but wasnt necessary

File: build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 plugins {
     id "com.matthewprenger.cursegradle" version "1.4.0"
-    id 'fabric-loom' version '0.2.4-SNAPSHOT'
+    id 'fabric-loom' version '0.2.7-SNAPSHOT'
```
Comment (Linguardium) [inline build.gradle:3]: probably could have put it at 2.6 but Fabric recommends 2.7

**Comment by TehNut** [comment]
Go ahead and update loader, bump yarn to the v2 spec, and bump the version in gradle.properties while you're at it.

**Comment by Linguardium** [comment]
done

**Pull request merged**